index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" type="image/x-icon" href="/assets/fabicon-removebg-preview.png">
+    <title>Youtube-like (TheOdinProject)</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <div class="menu">
+        <input type="checkbox" id="menu-toggle" />
+        <label for="menu-toggle" class="menu-button-container">
+          <div class="menu-button"></div>
+        </label>
+      </div>
+      <div class="logoCont">
+        <a href="https://www.youtube.com/">
+          <img id="logoImg" src="/assets/Youtube-logo.png" alt="youtube logo" />
+        </a>
+      </div>
+      <div class="searchbarCont">
+        <input type="text" placeholder="Search" />
+        <div class="srchIconCont">
+          <img
+            id="magGlass"
+            src="/assets/magnifying-glass-removebg.png"
+            alt=""
+          />
+        </div>
+        <div class="micCont">
+          <img id="mic" src="/assets/microphone-icon.png" alt="" />
+        </div>
+      </div>
+      <div class="buttonsCont">
+        <div class="crtIconCont">
+          <img id="crtImg" src="/assets/create-icon-removebg.png" alt="" />
+        </div>
+        <div class="bellIconCont">
+          <img id="bellIcon" src="/assets/bell-icon-removebg.png" alt="" />
+        </div>
+        <div class="prflImgCont">
+          <p id="profilePic">AR</p>
+        </div>
+      </div>
+    </header>
+    <div class="bodyCont">
+      <main>
+        <div class="mainCont">
+          <iframe
+            width="768"
+            height="480"
```

Comment:
- [ ] Great review [here](https://github.com/astroboyReloaded/Code-Review/pull/1/files?file-filters%5B%5D=.css&file-filters%5B%5D=.html&show-viewed-files=true#r1071385350). You were polite here. However, when you suggest a change, it is good communication to add reasons why you are suggesting this change. For example, in this case, you can highlight that it is industry best practice to not use inline styling but keep all CSS declarations in stylesheets. 👍 

#### When making the change to your inline comment, kindly edit the old one. Please don't create a new comment for it. 👍

Reply:
👍